Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -897,6 +901,9 @@ class BrowserViewController: UIViewController {
        }
 
     func finishEditingAndSubmit(_ url: URL, visitType: VisitType, forTab tab: Tab) {
+        if let query = self.searchController?.searchQuery {
+            self.appendQuery(query: query)
```

Comment:
I would rather call appendQuery from here https://github.com/cliqz/user-agent-ios/blob/develop/ReactNative/NativeModules/BrowserActions.swift#L38

Reply:
@chrmod  If I will move appentQuery the place which you mentioned it will work only when user will tap on a link from search controller and will not collect queries if user will tap on go button from keyboard.
In current behaviro we are collecting query for both cases.
Let me know which version you will prefer.